docs/core/proposer-based-timestamps.md

```diff
@@ -63,32 +62,36 @@ block.
 
 ### Timeliness
 
-When each validator on a Tendermint network receives a proposed block, it
+When each validator on the network receives a proposed block, it
 performs a series of checks to ensure that the block can be considered valid as
 a candidate to be the next block in the chain.
 
 The PBTS algorithm performs a validity check on the timestamp of proposed
 blocks. When a validator receives a proposal it ensures that the timestamp in
 the proposal is within a bound of the validator's local clock. Specifically, the
 algorithm checks that the timestamp is no more than `Precision` greater than the
-node's local clock and no less than `Precision` + `MessageDelay` behind than the
+node's local clock and no less than `Precision + MessageDelay` behind than the
 node's local clock. This creates range of acceptable timestamps around the
 node's local time. If the timestamp is within this range, the PBTS algorithm
 considers the block **timely**. If a block is not **timely**, the node will
-issue a `nil` `prevote` for this block, signaling to the rest of the network
+issue a `nil` prevote for this block, signaling to the rest of the network
 that the node does not consider the block to be valid.
 
 ### Clock Synchronization
 
-The PBTS algorithm requires the clocks of the validators on a Tendermint network
+The PBTS algorithm requires the clocks of the validators on the network
 are within `Precision` of each other. In practice, this means that validators
 should periodically synchronize to a reliable NTP server. Validators that drift
 too far away from the rest of the network will no longer propose blocks with
-valid timestamps. Additionally they will not view the timestamps of blocks
+valid timestamps. Additionally, they will not view the timestamps of blocks
 proposed by their peers to be valid either.
 
 ## See Also
 
-* [The PBTS specification](https://github.com/tendermint/tendermint/blob/master/spec/consensus/proposer-based-timestamp/README.md)
+* [The PBTS specification][pbts-spec]
  contains all of the details of the algorithm.
 
+[pbts-spec]: https://github.com/cometbft/cometbft/blob/main/spec/consensus/proposer-based-timestamp/README.md
+[consensus-parameters]: https://github.com/cometbft/cometbft/blob/main/spec/abci/abci%2B%2B_app_requirements.md#consensus-parameters
+[light-client-verification]: https://github.com/cometbft/cometbft/blob/main/spec/light-client/verification/README.md#failure-model
+[evidence-verification]: https://github.com/cometbft/cometbft/blob/main/spec/consensus/evidence.md#verification
```

Comment:
Can we set these with relative paths? This way we don't have to be changing the `.../main/...` part of the URL at every release.

Reply:
No, because they are on `spec/`. See: https://github.com/cometbft/cometbft-docs/issues/79